mesa_geo/geospace.py

```diff
@@ -203,24 +222,79 @@ def _recreate_rtree(self, new_agents=None):
         self.idx = index.Index(index_data)
         self.idx.agents = {id(agent): agent for agent in agents}
 
-    def update_bbox(self, bbox=None):
-        """Update bounding box of the GeoSpace."""
-        if bbox:
-            self.bbox = bbox
-        elif not self.agents:
-            self.bbox = None
+    def add_agents(self, agents):
+        """Add a list of GeoAgents to the layer without checking their crs.
```

Comment:
Nit: with the way this is phrased, I wouldn't be alerted that not checking for `crs` is dangerous. You might want to write the consequence here?

Reply:
Updated the docstring. Hopefully people (most likely myself) will be reminded : )